tests/unit/test_util.py

```diff
@@ -0,0 +1,23 @@
+from io import BytesIO, StringIO
+
+import pytest
+
+from vcr import request
+from vcr.util import read_body
+
+
+@pytest.mark.parametrize(
+    "input, output",
+    [
+        (BytesIO(b"Stream"), b"Stream"),
+        (StringIO("Stream"), b"Stream"),
+        (iter(["StringIter"]), b"StringIter"),
+        (iter([b"BytesIter"]), b"BytesIter"),
+        (iter([70, 111, 111]), b"Foo"),
+        ("String", b"String"),
+        (b"Bytes", b"Bytes"),
+    ],
+)
+def test_read_body(input, output):
+    r = request.Request("POST", "http://host.com/", input, {})
+    assert read_body(r) == output
```

Comment:
```suggestion
def test_read_body(input_, expected_output):
    r = request.Request("POST", "http://host.com/", input_, {})
    assert read_body(r) == expected_output
```

`input` name-collides with is a [Python built-in](https://docs.python.org/3/library/functions.html#built-in-functions).

At least one more cases like that in this pull requests.

Reply:
fixed, thanks (this was not failing on my computer).

Reply:
@sathieu it was working, yes, just not ideal. Thanks for the adjustments! :+1: